generate_core_dns_entry.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
same here, I suggest showing in the name that this script is only for AWS or implement it in a way more providers to be supported in the future.

Reply:
same as above.